client/elements/menus/language-menu.html

```diff
@@ -34,54 +34,121 @@
+        color: var(--paper-green-700)
+      }
+
       .hide-element {
         display: none;
       }
-    </style>
 
-    <paper-dropdown-menu class="custom" label="[[titleLabel]]" title="Select a translation in another language">
+      .iso-code {
+        color: var(--secondary-text-color);
+        @apply --paper-font-caption;
+        @apply --sc-sans-font;
+        position: absolute;
+        right: 15px;
+        margin-top: -34px;
+        margin-left: 10px;
+        text-transform: none;
+        pointer-events: none;
+      }
+
+      .language-choice-box:hover {
+        background-color: var(--paper-grey-200);
+        cursor: pointer;
+      }
+
+      .separator {
+        border: 0;
+        height: 0;
+        margin: 0;
+        border-top: 1px solid rgba(0, 0, 0, 0.1);
+        border-bottom: 1px solid rgba(255, 255, 255, 0.3);
+      }
+
+    </style>
+    <paper-dropdown-menu vertical-align="top" class="custom" label="[[titleLabel]]"
+                         title="Select a translation in another language" id="paper-dropdown-menu"
+                         dynamic-align=true>
       <paper-listbox slot="dropdown-content">
-        <!-- List the original language first if not already listed-->
-        <paper-item class$="orlang [[_computeHide(originalLanguage)]]"><a href="/[[inputUrl]]/[[originalLanguage]]">[[originalLanguage]]</a>
+        <paper-item class$="orlang [[_computeHide(originalLanguage)]]" title="Original language">
+          <a href="/[[inputUrl]]/[[originalLanguage]]">[[originalLanguage]]</a>
         </paper-item>
         <template is="dom-repeat" items="[[languageChoice]]" as="language">
           <template is="dom-if" if="[[!_otherLanguage(language, inputLanguage)]]">
-            <template is="dom-repeat" items="[[language.authors]]" as="author">
-              <!-- List all the authors and languages if they exist and does not list the ones that are from the input language-->
-              <paper-item id="[[language.id]]"><a href="/[[inputUrl]]/[[author]]">[[author]]</a></paper-item>
-            </template>
+            <hr class="separator">
+            <section class="language-choice-box">
+              <paper-item id='[[language.id]]' title="[[_languageTitle(language)]]">
+                <paper-item-body>
+                  <a href="/[[language.id]]/?author=[[language.author]]">[[_computeAuthorForList(language.author)]]</a>
+                </paper-item-body>
+              </paper-item>
+            </section>
+            <div class="iso-code">[[language.lang]]</div>
           </template>
         </template>
       </paper-listbox>
     </paper-dropdown-menu>
+
   </template>
 
   <script>
-      Polymer({
-          is: 'language-menu',
+      class SCLanguageMenu extends Polymer.Element {
+          static get is() {
+              return 'language-menu'
+          }
 
-          properties: {
+          static get properties() {
               // input of language-author array
```

Comment:
is this comment for the `languageChoice` array? If so, it should be lower. Also, I don't understand the comment (I know it's not yours).